azkaban-common/src/main/java/azkaban/server/AzkabanServer.java

```diff
@@ -118,17 +132,14 @@ private static Props loadConfigurationFromAzkabanHome() {
       logger.error("AZKABAN_HOME not set. Will try default.");
       return null;
     }
-
-    if (!new File(azkabanHome).isDirectory()
-        || !new File(azkabanHome).canRead()) {
+    if (!new File(azkabanHome).isDirectory() || !new File(azkabanHome).canRead()) {
```

Comment:
I think we need that. I didn't autoformat the entire file though - because we don't have a standard style yet. Once we establish that, then we can have a single format change and update everything. Also we can add style check validations to make our life simpler.